### PR TITLE
Add charter with membership requirements

### PR DESCRIPTION
**Pull request by tmandry**
- Create basic charter from content on home page
- Add membership requirements

### CODE REVIEW

**Comment by tmandry** [comment]
cc @rust-lang/wg-async-foundations

File: src/CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### Membership requirements
+
+Members are invited at the discretion of the working group leads, usually after a period of sustained contribution to the working group.
+
+In order to remain an active member, WG members must have active participation in last 6 months, where participation is one of the following:
+
+* Attended triage/sprint meetings
+* Opened PRs
+* Reviewed PRs
```
Comment (eminence) [inline src/CHARTER.md:50]: Presumably these PRs should be async-related PRs in either this repo, or in rust-lang/rust?   Would PRs in a repo like tokio be applicable here?

Comment (zeenix) [reply src/CHARTER.md:50]: I would think so. Tokio is an independent project.

**Review by nikomatsakis — APPROVED** [review]
LGTM

**Review by yoshuawuyts — APPROVED** [review]
Looks good! - I assume the plan is to as a next step to update [current published charter](https://www.rust-lang.org/governance/wgs/wg-async-foundations) by applying the requirements stated in the PR?

File: src/CHARTER.md (hunk 1 above)
Comment (tmandry) [reply src/CHARTER.md]: clarified

File: src/welcome.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
 # 👋🏽 Welcome
 
-Welcome to the wg-async-foundations website!
+Welcome to the home of wg-async-foundations! This working group is focused
+around implementation/design of the “foundations” for Async I/O.
```
Comment (eholk) [inline src/welcome.md:4]: We're broader than just Async I/O, right? To me not all async is I/O.

**Pull request merged**